rstudio/init.sh

```diff
@@ -0,0 +1,30 @@
+# download rstudio 
+wget http://download2.rstudio.org/rstudio-server-rhel-0.99.446-x86_64.rpm
+sudo yum install --nogpgcheck -y rstudio-server-rhel-0.99.446-x86_64.rpm
+
+# restart rstudio 
+rstudio-server restart 
+
+# add user for rstudio 
+adduser rstudio
+
+# create a Rscript that connects to Spark, to help starting user
+echo "cat('Now connecting to Spark for you.') 
```

Comment:
Actually since the code here doesn't depend on any env variables, can we just create a file `startSpark.R` here ? Then we can just do a `cp /root/spark-ec2/rstudio/startSpark.R /home/rstudio` here